docs/external_services/scrutinizer.md

```diff
@@ -0,0 +1,18 @@
+# Scrutinzer CI
+
+We use [Scrutinzer][scrutinizer] for coding standards and quality.
+
+To add your repository to be analysed:
+
+1. Visit [Scrutinzer][scrutinizer]
+1. Log-in via github
+1. Click `+ Add Repository`
+1. Select `graze` as the organisation
+    1. [graze only] If you do not have permissions, ask a graze/@open-source-team member for access
+1. Entry the repository name
+1. Select `PHP` as the language
+1. Ensure tests are not being run
```

Comment:
Why? (I assume this is because Travis is being used to run tests, but it would be good to mention it briefly)